packages/server/src/api/observation/observation.controller.ts

```diff
@@ -80,39 +79,30 @@ function hasReportingGroup(rosterInfo: RosterInfo, reportingGroup: string | unde
   return reportingGroup && rosterInfo.unit.org?.reportingGroup === reportingGroup;
 }
 
-function findReportSchema(reportSchemaId: string, orgId: number | undefined) {
+function findReportSchema(reportSchemaId?: string, orgId?: number) {
   return ReportSchema.findOne({
     relations: ['org'],
     where: {
-      id: reportSchemaId,
+      id: reportSchemaId || 'es6ddssymptomobs',
       org: orgId,
     },
   });
 }
 
-function rawReportToObservationApiModel(raw: ObservationApiRawReportModel): ObservationApiModel {
-  return {
-    edipi: raw.EDIPI,
-    timestamp: raw.Timestamp,
-    documentId: raw.ID,
-    reportSchemaId: raw.ReportType || 'es6ddssymptomobs',
-    unit: raw.Details?.Unit ?? '',
-    phoneNumber: raw.Details?.PhoneNumber ?? '',
-  };
-}
-
-function getReportingGroup(req: ApiRequest<EdipiParam, ObservationApiRawReportModel>) {
+function getReportingGroup(req: ApiRequest<EdipiParam, Report>) {
   return req.body.ReportingGroup || (req.body.Client?.Application && applicationMapping[req.body.Client?.Application]);
 }
 
-function saveObservationWithReportSchema(req: ApiRequest<EdipiParam, ObservationApiRawReportModel>, reportSchema: ReportSchema, manager: EntityManager) {
+function saveObservationWithReportSchema(req: ApiRequest<EdipiParam, Report>, reportSchema: ReportSchema, manager: EntityManager) {
   const observation = new Observation();
+  const { ID, EDIPI, ReportingGroup, ReportType, Timestamp, ...customColumns } = req.body;
   observation.documentId = req.body.ID;
   observation.edipi = req.body.EDIPI;
   observation.timestamp = timestampColumnTransformer.to(req.body.Timestamp);
   observation.reportSchema = reportSchema;
-  observation.unit = req.body.Details?.Unit ?? '';
+  observation.unit = `${req.body.Details?.Unit ?? ''}`;
   observation.reportingGroup = getReportingGroup(req);
+  observation.customColumns = customColumns as unknown as CustomColumns;
```

Comment:
@larskendall any thoughts on this? technically CustomColumns isn't the right type due to nested objects, and arrays

Reply:
Do you suppose we should flatten that data down before setting it on `observation.customColumns`? If you think we're ok to have nested data in there maybe we could just define CustomColumns as a recursive type sort of like this:

```typescript
export type CustomColumns = {
  [columnName: string]: ColumnValue | CustomColumns;
};
```

However, I'd probably lean toward flattening it if it doesn't cause issues, just because it seems like it'll be a headache to worry about nested types everywhere we interact with custom columns. We could potentially generate new keys for a flat object similarly to what the CSV export does where it just joins the nested keys with a `.`. We wouldn't lose any of the structure that way and could unflatten it any time we wanted.

Reply:
How would a flattened custom columns affect future creation of saved filters related to custom column content?  Say, I want to have a saved filter for all individuals that show a temperature greater than 100 ?  In my mind, this may be problematic when flattened?